### ISSUE TITLE
New-OVNetwork does not allow combining the parameters "vlanRange" and "NetworkSet"

### ISSUE THREAD

**Issue by mstreibe** [HewlettPackard/POSH-HPOneView#515]
### Expected Behavior
To create bulk networks and assign them to an existing network set, one should be able to specify a VLAN range and a network set to assign them to, but the cmdlet throws an error when trying this.

### Actual Behavior
```
New-OVNetwork : Parameter set cannot be resolved using the specified named parameters.
At line:1 char:1
+ New-OVNetwork -Name VLAN -VlanRange "2-2823,2826-3966" -Type Ethernet -Netwo ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [New-OVNetwork], ParameterBindingException
    + FullyQualifiedErrorId : AmbiguousParameterSet,New-OVNetwork
```

### Steps to reproduce
Execute this command:
`PS > New-OVNetwork -Name VLAN -VlanRange "100-2823,2826-3966" -NetworkSet "Production"`

### Version Information
```
HPE OneView PowerShell Library Version (`Get-HPOVVersion` or `$PSLibraryVersion`):
HPE OneView Appliance Version (`Get-HPOVVersion -ApplianceVer`):  
PS > Get-HPOVVersion

synergy.local                       LibraryVersion Path
--------------------                -------------- ----
ApplianceVersion: 5.40.00.423953.00 5.30.2560.2096 C:\Program Files\Hewlett Packard Enterprise\PowerShell\Modules\HP...
```

Output from `$PSVersionTable` on your Windows Host:

```
PS > $PSVersionTable

Name                           Value
----                           -----
PSVersion                      5.1.14409.1018
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.14409.1018
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

**Comment by ChrisLynchHPE** [comment]
This was addressed in the recent 5.00 and 5.20 library updates.  This will be fixed in the 5.30, 5.40 and 5.50 libraries soon.

**Comment by ChrisLynchHPE** [comment]
While this was fixed in prior 5.50 and 5.40 libraries, it is also within the following releases:

* [5.50.2680.1687](https://www.powershellgallery.com/packages/HPEOneView.550/5.50.2680.1687) 
* [5.40.2681.1725](https://www.powershellgallery.com/packages/HPEOneView.540/5.40.2681.1725)
* [5.30.2682.3811](https://www.powershellgallery.com/packages/HPEOneView.530/5.30.2682.3811)
* [5.20.2684.2161](https://www.powershellgallery.com/packages/HPOneView.520/5.20.2684.2161)
* [5.00.2685.1865](https://www.powershellgallery.com/packages/HPOneView.500/5.0.2685.1865)

Please confirm prior to closing this issue.

**Comment by ChrisLynchHPE** [comment]
Closing due to no further activity.  If you wish, feel free to re-open to discuss further.
